analytics/analytics.go

```diff
@@ -45,13 +47,26 @@ type dataBag struct {
 	Payload []byte `json:"payload"`
 }
 
-const kilobyte = 1024
-
-//HeartBeat is how often we send data to server, at the moment set to 15 Minutes
-var heartBeat = 15 * time.Minute
+type healthData struct {
+	NodeId 			string `json:"node_id"`
+	BTFSVersion 	string `json:"btfs_version"`
+	FailurePoint 	string `json:"failure_point"`
+}
 
 //Server URL for data collection
-var dataServeURL = "https://db.btfs.io/metrics"
+const (
```

Comment:
Do we need a switch to enable/disable report health data? For example, in my daily test daemon, I don't want this data send to postgres server.

Reply:
> Do we need a switch to enable/disable report health data? For example, in my daily test daemon, I don't want this data send to postgres server.

In the most case, the report-health functionality will not be triggered. so you don't need to worry about it in your daily test